types/next-auth/client.d.ts

```diff
@@ -28,6 +28,19 @@ interface SetOptionsParams {
     keepAlive?: number;
 }
 
+interface SignInObjectReturn {
```

Comment:
I suggest to use name similar to `next-auth` type to keep it consistent: `SignInResponse`

Source: https://github.com/nextauthjs/next-auth/blob/main/src/client/index.d.ts#L82